app/src/main/java/org/listenbrainz/android/ui/screens/brainzplayer/PlaylistScreen.kt

```diff
@@ -365,11 +366,13 @@ fun OnPlaylistClickScreen(playlistID: Long) {
                     Column(Modifier.padding(start = 10.dp)) {
                         Text(
                             text = it.title,
-                            color = androidx.compose.material3.MaterialTheme.colorScheme.onSurface
+                           // color = androidx.compose.material3.MaterialTheme.colorScheme.onSurface
+                            color= Color.White
```

Comment:
This fix is incorrect. We should never hardcode color values for an app which supports both light mode and dark mode.

Rather, we should look to make the black colored card as white or the relevant color to adapt automatically for light mode.

Reply:
Okay, get it. Thank You 😇